static/scss/partials/banners.scss

```diff
@@ -2,6 +2,10 @@
     text-align: center;
     position: relative;
     padding: $spacing-md;
+    
+    ul {
+        list-style-type: none;
```

Comment:
I feel that maybe this could be a `u-` utility style instead?

Reply:
I moved this statement here as the previous rule in `main.scss` was global. I wanted to avoid using a util class (and add additional markup) 

https://github.com/mozilla/fx-private-relay/blob/main/static/scss/partials/main.scss#L350-L352

Reply:
Got it!!